Source/AwsCommonRuntimeKit/Utilities.swift

```diff
@@ -41,6 +41,19 @@ extension String {
         let byteCursor = aws_byte_cursor_from_buf(output)
         return byteCursor.toData().base64EncodedString()
     }
+
+    public func base64EncodedSha256(allocator: Allocator = defaultAllocator, truncate: Int = 0) -> String {
+        let input: UnsafePointer<aws_byte_cursor> = fromPointer(ptr: self.awsByteCursor)
+        let emptyBuffer: UInt8 = 0
+        let bufferPtr: UnsafeMutablePointer<UInt8> = fromPointer(ptr: emptyBuffer)
+        let buffer = aws_byte_buf(len: 0, buffer: bufferPtr, capacity: 32, allocator: allocator.rawValue)
```

Comment:
Is it possible to use [AWS_SHA256_LEN](https://github.com/awslabs/aws-c-cal/blob/e364e9a42075447acf0ff9386a83c1ca19955b89/include/aws/cal/hash.h#L12) instead of `32` here? I don't know how accessible C stuff is from Swift

Reply:
It's accessible if it's a constant, not a macro

Reply:
updated :)